OWNERS

```diff
@@ -9,4 +9,5 @@ approvers:
   - christianvogt
   - andrewballantyne
   - spadgett
-component: Dynamic Plugin SDK
\ No newline at end of file
+  - vojtechszocs
```

Comment:
Ah good point! Sorry about that, I made up the owners files. We'll need to be sure we get work together on changes when new things happen or changes to existing things happen (that are public facing).